db/migrate/20210324091636_add_coordinates_to_groups.rb

```diff
@@ -0,0 +1,6 @@
+class AddCoordinatesToGroups < ActiveRecord::Migration[6.1]
+  def change
+    add_column :groups, :latitude, :float
```

Comment:
do we need this migration if we are searching mapbox by city rather than exact address?
I'm genuinely asking here

Reply:
I'm honestly not 100% sure. From following the Le Wagon slides I'm like 50/50 that Mapbox might be using it to geolocate. Again, not 100% on that, though.

Reply:
ok, let's keep them in for now